cups/dest.c

```diff
@@ -3502,6 +3502,12 @@ cups_enum_dests(
 
     num_dests = _cupsGetDests(http, IPP_OP_CUPS_GET_PRINTERS, NULL, &dests, type, mask);
 
+   /*
```

Comment:
OK, so this doesn't actually load all of the lpoptions information - we need to merge the root (/etc/cups/lpoptions) and user (~/.cups/lpoptions) instances and options, not just the user options (assuming that "filename" is initialized at this point...)